modules/kmm-validation-lifecycle.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * updating/kmm-preflight-validation.adoc
+
+:_content-type: CONCEPT
+[id="kmm-validation-lifecycle_{context}"]
+= Validation lifecycle
+
+Preflight validation attempts to validate every module loaded in the cluster. Preflight will stop running validation on a `Module`, once the validation is successful.
```

Comment:
No comma and s/once/after/:

"on a `Module` resource after"

And either "`Module` resource" (or whatever is accurate) if using the marked-up uppercase, or just "module" for a generic reference.